draft-ietf-tls-dtls13.md

```diff
@@ -956,6 +957,14 @@ configured not to perform a cookie exchange.  The default SHOULD be
 that the exchange is performed, however.  In addition, the server MAY
 choose not to do a cookie exchange when a session is resumed or, more
 generically, when the DTLS handshake uses a PSK-based key exchange.
+Servers which process 0-RTT requests and send 0.5-RTT responses
+without a cookie exchange risk being used in an amplification attack
+if the size of outgoing greatly exceeds the size of those that are received.
+A server SHOULD limit the amount of data it sends toward a client address
+before it verifies that the client is able to receive data at that address.
+A client address is valid after a cookie exchange or handshake completion.
+A server MAY apply a higher limit based on heuristics, such as if the client
+uses the same IP address as a recent connection.
```

Comment:
I wonder whether it makes sense to put out that this is not the regular amplification attack because the 0-RTT message is authenticated. While there is a risk of replay attack here, this assumes that 
(a) the attacker has been able to observe the initial 0-RTT request, or 
(b) the attacker is the endpoint transmitting the 0-RTT request itself. 

In case (b), if the endpoint was authenticated in the initial handshake then it is fairly easy to determine the source of the attack. In a classical amplification attack the adversary would probably also modify the source IP address to conceal its location, which does not help much here.  

Am I seeing this wrong?

Reply:
No I think you're right, but I think the text covers it reasonably well.